server/submission/resolvers.test.js

```diff
@@ -152,6 +150,113 @@ describe('Submission', () => {
     })
   })
 
+  describe('finshSubmission', () => {
+    function getInitialManuscript() {
+      // userId is defined in beforeEach
+      return {
+        submissionMeta: {
+          createdBy: userId,
+          stage: 'INITIAL',
+        },
+        // this is here just to fake upload process
+        files: [
+          {
+            url: 'fake-path.pdf',
+            name: 'FakeManuscript.pdf',
+            type: 'MANUSCRIPT_SOURCE',
+          },
+        ],
+      }
+    }
+    const fullManuscript = {
+      title: 'My Manuscript',
+      manuscriptType: 'research-article',
+      subjectAreas: ['cancer-biology'],
+      suggestedSeniorEditors: ['Senior 1', 'Senior 2'],
+      opposedSeniorEditors: [],
+      suggestedReviewingEditors: ['Editor 1', 'Editor 2'],
+      opposedReviewingEditors: [],
+      suggestedReviewers: [
+        { name: 'Reviewer 1', email: 'reviewer1@mail.com' },
+        { name: 'Reviewer 2', email: 'reviewer2@mail.com' },
+        { name: 'Reviewer 3', email: 'reviewer3@mail.com' },
+      ],
+      opposedReviewers: [],
+      noConflictOfInterest: false,
+      submissionMeta: {
+        coverLetter: 'my cover letter',
+        author: {
+          firstName: 'Firstname',
+          lastName: 'Lastname',
+          email: 'mymail@mail.com',
+          institution: 'Institution Inc',
+        },
+        hasCorrespondent: false,
+        discussedPreviously: false,
+        consideredPreviously: false,
+        cosubmission: false,
+      },
+    }
+    it('stores data to the backend with a new stage of QA', async () => {
+      const initialManuscript = getInitialManuscript()
+      const id = await save(manuscriptGqlToDb(initialManuscript, userId))
```

Comment:
Would it make sense to put this stuff in a `beforeEach` just for this `describe`?

Reply:
Yep

Reply:
Damn, now I need 2 reviews again :)